initfiles/componentfiles/configschema/templates/AddVaults.json

```diff
@@ -0,0 +1,102 @@
+{
+    "name": "Add Vault Feature",
+    "description": "Add necessary keys with empty values for vault feature",
+    "type": "modification",
+    "operations": [
+        {
+            "action": "delete",
+            "target_path": "/Environment/Software/vaults",
+            "data": {
+                "error_if_not_found": false
+            }
+        },
+        {
+            "action": "create",
+            "target_path": "/Environment/Software",
+            "data": {
+                "node_type": "vaults",
+                "save": {
+                    "name" : "vaults_node"
+                }
+            }
+        },
+        {
+            "action": "create",
+            "description": "Create the git element in Environment/Software/vaults",
+            "target_nodeid": "{{vaults_node}}",
+            "data": {
+                "node_type": "git",
+                "attributes": [
+                    {
+                        "name": "name",
+                        "value": "git_vault"
+                    },
+                    {
+                        "name": "url",
```

Comment:
Is it necessary that the url, and other keys that follow, have an entry? What would happen if the attributes were left out of the template since you are only using a (workaround) default value.

Or, based on a comment Tony just made, if the intent is for the user to fill it in, use a string indicating such.

If the attribute is not needed, maybe use "fill me in or remove if not needed"